Apricot.java

```diff
@@ -0,0 +1,17 @@
+public class Apricot {
+
+    private String color="";
+    static private int count=0; 
+    private float weight=0.0f;
+    public Apricot (Float InWeight, String InColor)
```

Comment:
better name for variables would be using first lowercase for example "color" or "inColor"